server/pom.xml

```diff
@@ -191,7 +196,8 @@
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-test</artifactId>
-            <version>${spring.version}</version>
+            <!-- needs this version for tests -->
```

Comment:
do we have an issue on spring version conflict?
should we had a item to review spring version everywhere?
It seems weird to have 2 versions on the same thing in the same project